conf/ushap.yaml

```diff
@@ -11,5 +11,7 @@ yearly:
   file_name : "USHAP_PM2.5_Y1K_{year}_V1.nc"
 
 layer: "PM2.5" #geographic weighted regression PM2.5
+rename_vars:
```

Comment:
Hey @KaurMahima,
I made a minimal change, I basically removed the colname that was hardcoded in the script